eis_toolkit/vector_processing/proximity_tool.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import geopandas as gpd
+from numbers import Number
+from beartype import beartype
+from beartype.typing import Union
+from rasterio import profiles, transform
+from eis_toolkit.vector_processing.distance_computation import distance_computation
+
+@beartype
+def calculate_proximity(geodataframe: gpd.GeoDataFrame, raster_profile: Union[profiles.Profile, dict], maximum_distance: Number) -> np.ndarray:
```

Comment:
This line, as some other lines, exceeds the allowed limit. If you use pre-commit locally, it will fix/point out these kind of cosmetic issues.

Also, perhaps it is better if the tool is named similarly as `distance_computation` is right now, so `proximity_computation`

Reply:
I had an issue with the pre-commit hooke
[pip is configured with locations that require TLS/SSL, however the ssl module in Python is not available](https://stackoverflow.com/questions/45954528/pip-is-configured-with-locations-that-require-tls-ssl-however-the-ssl-module-in)

However, is now resolved. I ran the precommit hooke and cleared the cosmetic problems
![image](https://github.com/user-attachments/assets/3e92aeb6-eedf-4fe5-9742-fdb4b78ee12d)